aerich/coder.py

```diff
@@ -17,7 +20,20 @@ def default(self, obj) -> Any:
             return super().default(obj)
 
 
+def load_index(obj: dict) -> Index:
+    """Convert a dict that generated by `Index.decribe()` to a Index instance"""
+    index = Index(fields=obj["fields"] or obj["expressions"], name=obj.get("name"))
+    if extra := obj.get("extra"):
+        index.extra = extra
+    if idx_type := obj.get("type"):
+        index.INDEX_TYPE = idx_type
+    return index
+
+
 def object_hook(obj) -> Any:
+    if "fields" in obj and not ({"expressions", "name", "type", "extra"} - obj.keys()):
```

Comment:
I'd suggest to move a set of keys to a `frozenset` constant

Reply:
Thanks for your suggestion~ 
I moved load_index logic out of `object_hook` for performance reasons.